airbyte-integrations/connectors/destination-snowflake/Dockerfile

```diff
@@ -7,9 +7,29 @@ FROM airbyte/integration-base-java:dev
 #  unzip /tmp/YourKit-JavaProfiler-2021.3-docker.zip -d /usr/local && \
 #  rm /tmp/YourKit-JavaProfiler-2021.3-docker.zip
 
+RUN yum install -y python3 python3-devel jq sshpass git && \
+  alternatives --install /usr/bin/python python /usr/bin/python3 60 && \
+  python -m ensurepip --upgrade && \
+  pip3 install dbt-snowflake==1.0.0
+
+# Luckily, none of normalization's files conflict with destination-bigquery's files :)
+# We don't enforce that in any way, but hopefully we're only living in this state for a short time.
+COPY --from=airbyte/normalization:dev /airbyte /airbyte
+# Install python dependencies
+WORKDIR /airbyte/base_python_structs
+RUN pip3 install .
+WORKDIR /airbyte/normalization_code
+RUN pip3 install .
+WORKDIR /airbyte/normalization_code/dbt-template/
+# Download external dbt dependencies
+# amazon linux 2 isn't compatible with urllib3 2.x, so force 1.26.15
+RUN pip3 install "urllib3<2"
```

Comment:
Maybe save a step and combine this with line 13 - combine pip installs into one layer?

Reply:
Wild guess this might be to avoid package version conflicts if there is a different version of `urllib3` already installed? Hence installing into a specific `WORKDIR`.

Reply:
Nah I just forgot this was an option :P worth testing to make sure it works though.